.github/workflows/build-and-publish-prebuilt.yaml

```diff
@@ -16,6 +16,9 @@ on:
       - "prebuilt-config/bitops-tag.yaml"
       - ".github/workflows/build-and-publish-prebuilt.yaml"
 
+  release:
+    types: [ published ]
```

Comment:
Looks like the omnibus image wasn't built on a release event, but instead the built was triggered when another "base" workflow updated the tag file.
With that approach there is no way to figure if it's a `release` or `pre-release` event here.